buildenv/jenkins/jobs/infrastructure/tagRepos

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+// TAG_NAME and TAG_ANNOTATION are passed in Jenkins parameters (Mandatory)
+// OPENJ9_SHA, OMR_SHA default to master/openj9 respectively if not set on Jenkins
+
+HTTP = 'https://'
+OMR_REPO = 'github.com/eclipse/openj9-omr.git'
+OPENJ9_REPO = 'github.com/eclipse/openj9.git'
+
+def clone_branch_push(REPO, TAG_NAME, TAG_ANNOTATION, TAG_POINT, POINT_TYPE) {
+    node ('worker') {
+        git "${HTTP}${REPO}"
+
+        echo "Check if TAG: '${TAG_NAME}' already exists"
+        // Return code needs to be 128
+        def STATUS
+        try {
+            STATUS = sh (
+                script: "git rev-parse --verify 'refs/tags/${TAG_NAME}'",
+                returnStatus: true
+            )
+        } catch(e) {
+            echo STATUS
+        }
+
+        def REPO_NAME = REPO.substring(REPO.lastIndexOf('/') +1, REPO.lastIndexOf('.git'))
+        echo "REPO_NAME: ${REPO_NAME}"
+
+        if (STATUS == 128) {
+            echo "Good, tag does not exist yet"
```

Comment:
Where is it documented that `128` is returned in this situation? Do we need to be specific here?

Reply:
I can't find it but I see 128 is used for other failure types too. So maybe we should just remove it and take any failure as "tag does not exist"